### PR TITLE
Snippet generator json fix

### PR DESCRIPTION
**Pull request by CodeGit**

### CODE REVIEW

File: src/snippets/template.py2.tmpl [hunk 1]
```diff
@@ -5,7 +5,7 @@ import sys, errno, re, json, ssl
 import urllib2
 from time import sleep
 
-BASE_URL = "<%= url %>?page_size=200<%= fileType === 'fasta' ? '&extra_fields=sequence' : '' %>"
+BASE_URL = "<%= url %>?page_size=100<%= fileType === 'fasta' ? '&extra_fields=sequence' : '' %>"
```
Comment (gustavo-salazar) [inline src/snippets/template.py2.tmpl]: Why are you reducing it to 100?

File: src/web-workers/download/index.worker.js [hunk 2]
```diff
@@ -31,7 +31,7 @@ const lut = new Map([
   ['fasta', 'text/x-fasta'],
   ['accession', 'text/plain'],
   ['json', 'application/json'],
-  ['ndjson', 'application/x-ndjson'],
+  // ['ndjson', 'application/x-ndjson'],
```
Comment (gustavo-salazar) [inline src/web-workers/download/index.worker.js]: Why to take ndjson out of the list?

File: src/components/DownloadForm/index.js [hunk 3]
```diff
@@ -273,33 +272,44 @@ class DownloadForm extends PureComponent {
...
-              <Estimate data={data} />
-              {/* Only display if the response from the API is a list of items */}
-              {description[main].db && !description[main].accession && (
-                <Snippet fileType={fileType} url={endpoint} subset={subset} />
-              )}
+          {({ data, download, isStale }) => {
+            const count = (data.payload && data.payload.count) || 0;
+            const { db, integration } = description[main];
+            const noData = count === 0 && (db !== null || integration !== null);
+            console.log(`NODATA DB: ${db} Integration:${integration}`);
```
Comment (gustavo-salazar) [inline src/components/DownloadForm/index.js]: Was this left by mistake? or do you actually want a log here

File: src/snippets/template.py2.tmpl (hunk 1 above)
Comment (CodeGit) [reply src/snippets/template.py2.tmpl]: Probably just a leftover from testing, I might have been testing with 10 and just added '0' to my test.

**Pull request merged**